helpers/metadata_test.go

```diff
@@ -0,0 +1,81 @@
+package helpers
+
+import (
+	tektonv1beta1 "github.com/tektoncd/pipeline/pkg/apis/pipeline/v1beta1"
+	"testing"
+)
+
+var (
+	testObject      = &tektonv1beta1.PipelineRun{}
+	testAnnotations = map[string]string{
+		"a.test/annotation":              "foo",
+		"another.test/annotation-string": "bar",
+	}
+	testLabels = map[string]string{
+		"a.test/label":              "foo",
+		"another.test/label-string": "bar",
+	}
+)
+
+func init() {
+	testObject.SetAnnotations(testAnnotations)
+	testObject.SetLabels(testLabels)
+}
+
+func TestHasAnnotation(t *testing.T) {
+	checkExistingAnnotation := HasAnnotation(testObject, "a.test/annotation")
+	if checkExistingAnnotation != true {
```

Comment:
Applies to others, but in short:
```suggestion
	if !checkExistingAnnotation {
```
Variable name could be renamed to something that doesn't read like an action. Something like `hasAnnotation` or `exists`. That way the `if` condition would read better.